### PR TITLE
8131 rename trace and bundleproperty in case/uco code

### PR DESCRIPTION
**Pull request by markmckinnon**

### CODE REVIEW

File: case-uco/java/src/org/sleuthkit/caseuco/CaseUcoExporter.java [hunk 1]
```diff
@@ -590,13 +590,13 @@ private void assembleGenInfo(String uuid, BlackboardArtifact artifact, List<Json
     }
 
     private void assembleWebHistory(String uuid, BlackboardArtifact artifact, List<JsonElement> output) throws TskCoreException {
-        Trace userNameNode = new BlankTraceNode();
+        ObservableObject userNameNode = new BlankTraceNode();
```
Comment (bcarrier) [inline case-uco/java/src/org/sleuthkit/caseuco/CaseUcoExporter.java:593]: @markmckinnon We should probably rename BlankTraceNode to BlankObservableObject.

File: case-uco/java/src/org/sleuthkit/caseuco/UcoObject.java [hunk 2]
```diff
@@ -27,6 +27,8 @@
  */
 abstract class UcoObject {
 
+    static final String UCO_OBJECT = "uco-observable:";
```
Comment (bcarrier) [inline case-uco/java/src/org/sleuthkit/caseuco/UcoObject.java:30]: I'd name this something like UCO_OBSERV.  I think we're going to have several of these prefixes.

File: case-uco/java/src/org/sleuthkit/caseuco/CaseUcoExporter.java (hunk 1 above)
Comment (markmckinnon) [reply case-uco/java/src/org/sleuthkit/caseuco/CaseUcoExporter.java]: Changed and commited.

File: case-uco/java/src/org/sleuthkit/caseuco/UcoObject.java (hunk 2 above)
Comment (markmckinnon) [reply case-uco/java/src/org/sleuthkit/caseuco/UcoObject.java]: Changed and commited

**Review by rcordovano — APPROVED** [review]

**Pull request merged**